src/components/SdkGroup.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
Going ahead with "Item", "ItemGroup", and "ItemGroupsList", since these are generic components and may be reused elsewhere going forward, irrespective of whether it's an SDK

Reply:
Thinking of going ahead with "Book", "BookGroup and "BookGroupsList"

Reply:
Sounds good.

Reply:
Done: 696e4a5